refresh/fromswagger/swaggerize.js

```diff
@@ -87,10 +84,6 @@ function parseSwagger (api) {
 
   Object.keys(api.paths).forEach(function (path) {
     var resource = genUtils.resourceNameFromPath(path)
-    if (resource === '*') {
-      // ignore a resource of '*' as a default route for this is set up in the template.
-      return
-    }
```

Comment:
Can you explain briefly why this is okay to remove? Does the validator ensure this is never the case?